website/package.json

```diff
@@ -41,7 +41,7 @@
     "remark-math": "3",
     "semantic-release": "^19.0.3",
     "use-editable": "^2.3.3",
-    "z3-solver": "4.11.0"
+    "z3-solver": "4.12.0"
```

Comment:
it is 4.12.1 already, but it seems to require an update to yarn.lock as well, which takes a different command.
I guess the upgrade script wasn't fully tested after z3-solver got added to package.json

Reply:
the main branch has this issue too

Reply:
I will try to resolve this